### PR TITLE
New footer

### PR DESCRIPTION
**Pull request by lpil**
foot-kitten

### CODE REVIEW

File: shared/components/SiteFooter/index.js [hunk 1]
```diff
@@ -1,73 +1,144 @@
...
+function Banner({ page }) {
+  if (page === 'Community') {
+    return (
+      <div className="SiteFooter__banner SiteFooter__conference block">
+        <h2 className="SiteFooter__banner-title">
+          React London 2017
+        </h2>
+        <p>
+          <span className="text-background">
+            We're launching a new conference<br />focused on React
```
Comment (pixelblend) [inline shared/components/SiteFooter/index.js]: This is my fault :(

Should probably be `&rsquo;` rather than `'`

**Comment by pixelblend** [comment]
👍 , apart from my HTML entity snafu

File: shared/components/SiteFooter/index.js (hunk 1 above)
Comment (lpil) [inline shared/components/SiteFooter/index.js]: Thanks!

File: shared/components/SiteFooter/index.js [hunk 2]
```diff
@@ -1,73 +1,144 @@
...
+      <div className="SiteFooter__banner SiteFooter__conference block">
+        <h2 className="SiteFooter__banner-title">
+          React London 2017
+        </h2>
+        <p>
+          <span className="text-background">
+            We&rsquo;re launching a new conference<br />focused on React
+          </span>
+        </p>
+        <a className="SiteFooter__banner-cta" href="#">
```
Comment (AndrewBestbier) [inline shared/components/SiteFooter/index.js]: Shouldn't this link?

File: shared/components/SiteFooter/index.js [hunk 3]
```diff
@@ -1,73 +1,144 @@
...
+    <div className="SiteFooter__banner SiteFooter__community block">
+      <h2 className="SiteFooter__banner-title">
+        React London Meetup
+      </h2>
+      <p>
+        <span className="text-background">
+          Looking for our meetups?
+        </span>
+      </p>
+      <a className="SiteFooter__banner-cta" href="#">
```
Comment (AndrewBestbier) [inline shared/components/SiteFooter/index.js]: See above, should link?

File: shared/components/SiteFooter/index.js [hunk 4]
```diff
@@ -1,73 +1,144 @@
...
+          <object
+            className="SiteFooter__badger"
+            data="/img/SVG/Badger.svg"
+            type="image/svg+xml"
           />
-        </object>
-      </a>
 
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
```
Comment (AndrewBestbier) [inline shared/components/SiteFooter/index.js]: We need the copy before we merge this into master

File: shared/components/Community/index.js [hunk 5]
```diff
@@ -9,6 +9,16 @@ import MailingList from '../MailingList';
 import JoinSlack from '../JoinSlack';
 import FutureEvents from '../FutureEvents';
 
+const usefulLinks = [
```
Comment (AndrewBestbier) [inline shared/components/Community/index.js]: Nice :) These work perfectly when given data. Whats the plan with these when they write the copies, new pages from UX and Design?

**Comment by AndrewBestbier** [comment]
Other than a few small comments looks good :) Also merge conflict

File: shared/components/Community/index.js (hunk 5 above)
Comment (lpil) [inline shared/components/Community/index.js]: Dunno to be honest.

**Pull request merged**